globus_automate_client/cli/main.py

```diff
@@ -6,6 +6,12 @@
 from globus_automate_client.cli import actions, flows, queues
 from globus_automate_client.token_management import CONFIG_PATH
 
+# Monkey patching out the unsafe load capability
+# Only use safe_load for our purposes
+import yaml
+del yaml.unsafe_load
```

Comment:
Never saw this approach before. I like the ~paranoid~ defensive coding approach.